controllers/kubeadmconfig_controller.go

```diff
@@ -68,6 +68,7 @@ func ClusterCertificatesSecretName(clusterName string) string {
 // +kubebuilder:rbac:groups=bootstrap.cluster.x-k8s.io,resources=kubeadmconfigs/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=clusters;machines,verbs=get;list;watch
 // +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;create;update;patch
+// +kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;patch
```

Comment:
We can merge this line with the one above, reducing the amount of rules generated in the rbac file
```golang
// +kubebuilder:rbac:groups="",resources=secrets;events,verbs=get;list;watch;create;update;patch
```

Reply:
Good idea! I have updated it. @vincepri